src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -565,6 +566,46 @@ const DECISION_REVIEW_CONTENT = (
   </div>
 );
 
+/**
+ * Translates an array of two ints into a string that conveys a duration estimate
+ * @typedef {Object} durationText contains strings to fill in time snippets in NextEvents
+ * @property {string} header formatted time string to be used in the duration card header
+ * @property {string} description formatted time string to be used in the duration card description
+ * @param {number[]} timeliness two integers that represent the low and high time durations
+ * (in months) of a given thing
+ * @returns {durationText} formatted to convey the estimated duration range, in months
+ */
+export const makeDurationText = (timeliness) => {
+  const durationText = {
+    header: 'unknown',
+    description: 'unknown',
+  };
+
+  if (!timeliness || !Array.isArray(timeliness) || timeliness.length !== 2) {
+    const durationError = new Error(
+      'vets_appeals_v2_helpers_makeDurationText_bad_timeliness_input'
+    );
+    Raven.captureException(durationError);
+    return durationText;
+  }
+
+  const lowEst = timeliness[0];
+  const highEst = timeliness[1];
+  const estIsExact = (lowEst === highEst);
+
+  if (estIsExact && lowEst === 1) {
+    durationText.header = '1 month';
+    durationText.description = 'about 1 month';
+  } else if (estIsExact) {
+    durationText.header = `${lowEst} months`;
+    durationText.description = `about ${lowEst} months`;
+  } else {
+    durationText.header = `${lowEst}–${highEst} months`;
+    durationText.description = `between ${lowEst}–${highEst} months`;
```

Comment:
Thanks for the en-dash, but the description should use the word "and." :)

```
`between ${lowEst} and ${highEst} months`
```

Reply:
Ah, right, thanks!

Reply:
Updated: https://github.com/department-of-veterans-affairs/vets-website/pull/7209/files#diff-07c91cd8e7a0729551ae3b4b7a393441R604